src/components/common/Breadcrumbs.tsx

```diff
@@ -57,33 +57,28 @@ const Breadcrumbs = (props: BreadcrumbsPropsDefault) => {
           </ol>
         </nav>
       )}
-    </>
+    </ErrorBoundaryWithAnalytics>
   );
 };
 
 interface BreadcrumbProps {
   name: string;
   slug?: string;
   index: number;
-  addAnalytics?: boolean;
```

Comment:
analytics should be opt out I think? We probably want them on in 99% of sites right?